app/components/ViewEntries.js

```diff
@@ -4,6 +4,7 @@ import AuthEntry from './AuthEntry';
 const ViewEntries = ({ entries, onEdit, onRemove, onCopyToken }) => (
   <div className="auth-list">
     {entries.map((entry, idx) => (
+      entry.filtered && 
```

Comment:
`entry.filtered` will be `undefined` when you haven't used the search bar. All entries will be hidden when you open a note until you search for anything.

Reply:
Good! Thank you @johnny243, must have missed that. I just committed the change in logic according to the my previous change of handling the filter only during search. 

What do you think?